sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/KalixClient.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.impl
+
+import com.google.protobuf.ExperimentalApi
+import org.springframework.http.{ HttpHeaders, MediaType }
+import org.springframework.web.reactive.function.client.WebClient
+
+import java.util.concurrent.CompletionStage
+import scala.concurrent.{ Future, Promise }
+import scala.jdk.FutureConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+trait KalixClient {
+  //def post[P, R](uri: String, body: P, returnType: Class[R]): DeferredCall[P, R]
+  //def get[R](uri: String, resp: Class[R]): DeferredCall[_, R] // FIXME not sure what to use by default
+
+  @ExperimentalApi
+  def postCall[P, R](uri: String, body: P, returnType: Class[R]): CompletionStage[R]
+
+  @ExperimentalApi
+  def getCall[R](uri: String, resp: Class[R]): CompletionStage[R]
```

Comment:
Now that I think about it one more time, maybe we can have already a `post` and `get` returning a `DeferredCall`. 

But instead of returning `DeferredCallImpl` we return `RestDeferredCallImpl`. The `execute` method for `RestDeferredCallImpl` we run the web client call. 

If we take that path, we won't break any code. The API will be 'stable' already. We can still mark them with `@ApiMayChange` though. 

My point is, we will be able to cut a release for now, even if the transcoding is not yet in place. Obviously, `forwards` and `timers` won't work. We will need to have some clear error message about it. Also document that they are not yet supported.

Reply:
I see what you mean, that would be better indeed. Let me try

Reply:
Updated in last version.